jwst/regtest/test_fgs.py

```diff
@@ -0,0 +1,88 @@
+"""Regression tests for FGS Guidestar in ID and FINEGUIDE modes"""
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.lib.suffix import replace_suffix
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+def is_like_truth(rtdata, fitsdiff_default_kwargs, suffix, truth_path='truth/fgs/test_fgs'):
+    """Compare step outputs with truth"""
+    output = replace_suffix(
+        os.path.splitext(os.path.basename(rtdata.input))[0], suffix
+    ) + '.fits'
+    rtdata.output = output
+
+    rtdata.get_truth(os.path.join(truth_path, output))
+
+    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    assert diff.identical, diff.report()
+
+
+@pytest.fixture(scope='module')
+def run_fgs_imaging_pipelines(jail, rtdata_module):
```

Comment:
We didn't really need a test of the imaging (science) mode data, but it's OK to have it.

Reply:
Part of my ignorance of FGS terminology. Realized later that the specification of GUIDESTAR does not, in fact, include the science side of things.